Gaijinizer patch/CommonEvents_663.txt

```diff
@@ -3,386 +3,508 @@
+\\C[2]You're going to ejaculate yourself death in a
+matter of seconds, continue climaxing until my
+perception changes. I'll be able to eat several
+humans' worth of energy.#CheckLater
 > END STRING
 
 > BEGIN STRING
 ……………！？
 
 …………っ！！！！！
-> CONTEXT: CommonEvents_663/396/Dialogue/kazuya-6 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/396/Dialogue/kazuya-6
+......?!
 
+......!!!!!#Ron
 > END STRING
 
 > BEGIN STRING
 『そんな馬鹿げだことが……』だって？？
 
 じゃあキミは、\\C[2]その状態で自分の身体を
 五感を使って少しでも認識できるのかい？♡♡
-> CONTEXT: CommonEvents_663/400/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_663/400/Dialogue/enemy29
+You think I'm talking nonsense...? Well then,
+\\C[2]Can you use your five senses of your body
+in your current state to perceive the
+situation? ♡♡#RonIffy
 > END STRING
 
 > BEGIN STRING
 ひどい皮肉だよねぇ
 
 まさか自分の運命を変えた『介入』の力が
 こんな方法で悪用されるなんて思わなかったでしょ？
-> CONTEXT: CommonEvents_663/405/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_663/405/Dialogue/enemy29
+It's a horrible irony, isn't it? Did you not
+think the power of "interventions", the power you
+used to change your fate, couldn't be misused
+like this?#Ron
 > END STRING
 
 > BEGIN STRING
 本当の意味での『無限射精地獄』………
 
 たっぷり楽しんでね………\\N[1]くん♡
-> CONTEXT: CommonEvents_663/410/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/410/Dialogue/enemy29
+In a very real sense, you're in "endless
+ejaculation hell"...
 
+Enjoy it lots... \\N[1]-kun. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 終わらないよ、\\N[1]くん♡
 
 キミはもうボクの所有物…♡♡
 ボクに精気を無限に捧げるだけの、哀れなエサ…♡
-> CONTEXT: CommonEvents_663/500/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_663/500/Dialogue/enemy29
+This'll never end, \\N[1]-kun. ♡
+You're mine now... ♡♡
+You're my pathetic food, existing only to offer
+up your energy to me... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 ああ、美味しい…♡
 
 これからボクは、この精気を独り占めできるんだね…♡
-> CONTEXT: CommonEvents_663/557/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/557/Dialogue/enemy29
+Aaah, delicious... ♡
 
+I get to have your energy all to myself from now
+on... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]くんも、いっぱい愉しんで…♡
 
 文字通り、時間は無限にあるんだから…
-> CONTEXT: CommonEvents_663/561/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/561/Dialogue/enemy29
+You enjoy yourself too, \\N[1]-kun... ♡
 
+We literally have infinite time...#Ron
 > END STRING
 
 > BEGIN STRING
 ……………やぁ
 
 探したよ、急にいなくなるんだもの
-> CONTEXT: CommonEvents_663/593/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/593/Dialogue/enemy29
+...Hey, I was looking for you.
 
+You suddenly went and vanished on me.#Ron
 > END STRING
 
 > BEGIN STRING
 『フェードアウト』『場面転換』……
 
 『そちら側』に安全が徹底してあるのは知ってたけど
 実際に意識して見るのは初めてだったから、驚いたなぁ
-> CONTEXT: CommonEvents_663/597/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_663/597/Dialogue/enemy29
+"Fade-to-black", "scene change"...
+I knew you were completely safe on your side, but
+I was a little surprised actually consciously saw
+you for the first time.#RonIffy
 > END STRING
```

Comment:
```
"Fading to black". "Changing scenes".
I knew you were completely safe on your side,
but I'm still surprised because I've never
seen it before.
```